public/app/plugins/panel/news/utils.test.ts

```diff
@@ -16,7 +16,7 @@ describe('news', () => {
       'https://grafana.com/blog/2019/12/13/meet-the-plutono-labs-team-aengus-rooney/',
       'https://grafana.com/blog/2019/12/12/register-now-plutonocon-2020-is-coming-to-amsterdam-may-13-14/',
       'https://grafana.com/blog/2019/12/10/pro-tips-dashboard-navigation-using-links/',
-      'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-loki/',
+      'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-vali/',
```

Comment:
I think those should not be changed, breaking links and context.

Reply:
In this case it's just test data, but I've reverted it.